fab/utils.py

```diff
@@ -11,7 +11,7 @@
 import re
 from getpass import getpass
 
-from github import Github, InputGitAuthor
+from github3 import login
```

Comment:
Does github3 need to go back into requirements.txt, a la https://github.com/dimagi/commcare-hq/pull/18232/files#diff-444e80f30bd2f1e25b779966c9621b9bL90 ?

Reply:
yep: https://github.com/dimagi/commcare-hq/pull/18646